drivers/spi/spi_max32.c

```diff
@@ -242,19 +275,46 @@ static int spi_max32_transceive(const struct device *dev)
 	dfs_shift = spi_max32_get_dfs_shift(ctx);
 
 	len = spi_context_max_continuous_chunk(ctx);
+#ifdef CONFIG_SPI_RTIO
+	if (sqe->op == RTIO_OP_TINY_TX) {
+		len = sqe->tiny_tx.buf_len;
+		data->req.txData = (uint8_t *)sqe->tiny_tx.buf;
+		data->req.rxData = 0;
+	} else {
+		len = sqe->txrx.buf_len;
+		data->req.txData = (uint8_t *)sqe->txrx.tx_buf;
+		data->req.rxData = sqe->txrx.rx_buf;
+		data->req.rxData = sqe->txrx.rx_buf;
```

Comment:
Doing this twice?

Reply:
This was done in the original transceive (line 295-297), I thought it was redundant but will remove it